src/components/grid/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import PropTypes from "prop-types"
+import classNames from "classnames"
+
+import styles from "./styles.module.scss"
+
+const Grid = ({ alignment, children, layout }) => {
+  const classes = classNames(styles.grid, {
+    [styles[`alignment_${alignment}`]]: alignment,
+    [styles[`layout_is_${layout}`]]: layout
+  })
+
+  return <div className={classes}>{children}</div>
+}
+
+Grid.propTypes = {
+  /**
+   * Specifies how each grid item should be aligned within the grid
+   */
+  alignment: PropTypes.oneOf(["top", "bottom", "center"]),
+  /**
+   * Specifies anything between the opening and closing tag.
+   */
+  children: PropTypes.node,
+  /**
+   * Specifies the width of each grid item within the grid on larger screens
+   */
+  layout: PropTypes.oneOf(["one_third", "one_fourth", "one_half"])
```

Comment:
I think we should either use fractions or quicker words. Meaning `quarter` or `1/4` over `one_fourth`. Not sure why, though.

Reply:
I'm taking this to general assembly! We'll see what happens. 😬

Reply:
I think we should change `layout` to `col_width` and use fractions as the accepted values, e.g. `1/3`, `1/4`, etc.

That isn't perfect, but I think it's a little more semantic and easier to scale (for now).